applications/appunorganized/appunorganized-repository/category.xml

```diff
@@ -3,15 +3,6 @@
    <feature id="org.csstudio.email.feature">
       <category name="org.csstudio.category.applications"/>
    </feature>
-   <feature id="org.csstudio.pvmanager.ca.feature">
-      <category name="org.csstudio.category.applications"/>
-   </feature>
-   <feature id="org.csstudio.utilities.feature">
-      <category name="org.csstudio.category.applications"/>
-   </feature>
-   <feature id="org.csstudio.utilities.rap.feature">
-      <category name="org.csstudio.category.applications"/>
-   </feature>
```

Comment:
@shroffk I removed these features:
pvmanager.ca
utilities
utilities.rap
utilities has graphene plugins.  We should make a new feature with graphene, named as such.